common/production_methods/babylon_urban.txt

```diff
@@ -1,44 +1,254 @@
+			goods_input_coal_add = 5
+			goods_output_transportation_add = 15
+		}
+
+		level_scaled = {
+			building_employment_engineers_add = 100
+			building_employment_machinists_add = 400
+			building_employment_laborers_add = 1000
+		}
+	}
+}
+
+pm_steam_buses = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		threshing_machine
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 25
+		}
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_coal_add = 5
+			goods_input_engines_add = 5
+			# output goods
+			goods_output_transportation_add = 20
+		}
+
+		level_scaled = {
+			building_employment_engineers_add = 250
+			building_employment_machinists_add = 250
+			building_employment_laborers_add = 1000
 		}
 	}
-	country_modifiers = {
-		unscaled = {
-			state_migration_pull_mult = 0.1
+}
+
+pm_motor_buses = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		combustion_engine
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 50
 		}
 	}
 
 	building_modifiers = {
 		workforce_scaled = {
-			goods_input_automobiles_add = 1
+			goods_input_automobiles_add = 2
+			goods_input_oil_add = 5
 			# output goods
-			goods_output_transportation_add = 5
+			goods_output_transportation_add = 25
 		}
 
 		level_scaled = {
-			building_employment_laborers_add = -100
+			building_employment_laborers_add = 1000
+			building_employment_engineers_add = 500
 		}
 	}
 }
 
-pm_no_automobile_roads = {
+##Road Transport
+
+pm_railway_prioritization = {
+	texture = "gfx/interface/icons/production_method_icons/no_public_transport.dds"
+}
+
+pm_hand_carts = {
 	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		urbanization
+	}
 	state_modifiers = {
 		workforce_scaled = {
 			state_pollution_generation_add = 0
+			state_infrastructure_add = 4
+		}
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_fabric_add = 5
+			goods_input_wood_add = 5
+			# output goods
+			goods_output_transportation_add = 10
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = 2500
+			building_employment_shopkeepers_add = 500
+		}
+	}
+}
+
+pm_wagons = {
+	texture = "gfx/interface/icons/production_method_icons/public_trams.dds"
+	unlocking_technologies = {
+		urban_planning
+	}
+	state_modifiers = {
+		workforce_scaled = {
+			state_pollution_generation_add = 5
+			state_infrastructure_add = 8
+		}
+	}
+
+	building_modifiers = {
+		workforce_scaled = {
+			goods_input_fabric_add = 5
+			goods_input_wood_add = 5
+			goods_input_grain_add = 10
+			# output goods
+			goods_output_transportation_add = 15
+		}
+
+		level_scaled = {
+			building_employment_laborers_add = 2000
+			building_employment_farmers_add = 500
+			building_employment_shopkeepers_add = 500
 		}
 	}
+}
+
+pm_steam_trucks = {
```

Comment:
I thought we didn't want to replace railways?

Reply:
I know the temptation of adding more high tech stuff, but this is too much infrastructure.

Reply:
By this point better railway PMs are also available at t3 and t4 techs. I can nerf the infra values more however, down to 2.5/5/7.5/10 instead of the current 4/8/12/15

Reply:
I would freeze the infrastructure after 8

Reply:
I've frozen the infrastructure after 8 but made the later PMs more cost effective in terms of money per point of infrastructure. 37.5 money per point of infrastructure at t3 and 32.5 at t4.

Reply:
nice